README.md

```diff
@@ -1,91 +1,135 @@
-# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;cuSpatial - GPU-Accelerated Spatial and Trajectory Data Management and Analytics Library</div>
+# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;cuSpatial - GPU-Accelerated Vector Geospatial Data Analysis</div>
 
-**NOTE:** cuSpatial depends on [cuDF](https://github.com/rapidsai/cudf) and
-[RMM](https://github.com/rapidsai/rmm) from [RAPIDS](https://rapids.ai/).
+> **Note** 
+> 
+> cuSpatial depends on [cuDF](https://github.com/rapidsai/cudf) and [RMM](https://github.com/rapidsai/rmm) from [RAPIDS](https://rapids.ai/).
 
-## Operations
+## Resources
 
-cuSpatial supports the following operations on spatial and trajectory data:
+- [cuSpatial User's Guide](https://docs.rapids.ai/api/cuspatial/stable/user_guide/cuspatial_api_examples.html): Python API reference and guides
+- [cuSpatial Developer Documentation](https://docs.rapids.ai/api/cuspatial/stable/developer_guide/index.html): Understand cuSpatial's architecture
+- [Getting Started](https://rapids.ai/start.html): Instructions for installing cuSpatial
+- [cuSpatial Community](https://github.com/rapidsai/cuspatial/discussions): Get help, collaborate, and ask the team questions
+- [cuSpatial Issues](https://github.com/rapidsai/cuspatial/issues/new/choose): Request a feature/documentation or report a bug
+- [cuSpatial Roadmap](https://github.com/orgs/rapidsai/projects/41/views/5): Report issues or request features.
 
-1. Spatial window query
-2. Point-in-polygon test
-3. Haversine distance
-4. Hausdorff distance
-5. Deriving trajectories from point location data
-6. Computing distance/speed of trajectories
-7. Computing spatial bounding boxes of trajectories
-8. Quadtree-based indexing for large-scale point data
-9. Quadtree-based point-in-polygon spatial join
-10. Quadtree-based point-to-linestring nearest neighbor distance
-11. Distance computation (point-point, point-linestring, linestring-linestring)
-12. Finding nearest points between point and linestring
+## Overview
+Now based on the [GeoArrow](https://github.com/geoarrow/geoarrow) format, cuSpatial accelerates vector geospatial operations through GPU parallelization. As part of the RAPIDS libraries, it's inherently connected to [cuDF](https://github.com/rapidsai/cudf), [cuML](https://github.com/rapidsai/cuml), and [cuGraph](https://github.com/rapidsai/cugraph), enabling GPU acceleration across entire workflows. 
```

Comment:
"Now based on" is "news", so it will grow stale. Let's move the GeoArrow mention down closer to the GeoPandas mention.

```suggestion
cuSpatial accelerates vector geospatial operations through GPU parallelization. As part of the RAPIDS libraries, cuSpatial is inherently connected to [cuDF](https://github.com/rapidsai/cudf), [cuML](https://github.com/rapidsai/cuml), and [cuGraph](https://github.com/rapidsai/cugraph), enabling GPU acceleration across entire workflows. 

cuSpatial represents data in [GeoArrow](https://github.com/geoarrow/geoarrow) format, which enables compatibility with the [Apache Arrow](https://arrow.apache.org) ecosystem.
```